data/Dockerfiles/acme/docker-entrypoint.sh

```diff
@@ -18,18 +18,14 @@ while true; do
 		-f ${ACME_BASE}/acme/private/account.key \
 		-k ${ACME_BASE}/acme/private/privkey.pem \
 		-c ${ACME_BASE}/acme \
-		$*
+		${MAILCOW_HOSTNAME} autoconfig.${MAILCOW_HOSTNAME#*.} autoconfig.${MAILCOW_HOSTNAME#*.} ${ADDITIONAL_SAN}
 
 	case "$?" in
 		0) # new certs
 			# cp the new certificates and keys 
 			cp ${ACME_BASE}/acme/fullchain.pem ${ACME_BASE}/cert.pem
 			cp ${ACME_BASE}/acme/private/privkey.pem ${ACME_BASE}/key.pem
 
-			# also generate new dhparams with new certs
-			#(on the first run, the old dhparams may be the fakeoil from git)
-			openssl dhparam -out ${ACME_BASE}/dhparams.pem 2048
```

Comment:
@andryyy Could you please elaborate this?

In contrast to `key.pem`, `dhparams.pem` can be public. So it's ok to use the ones from the git-repository. But is it desirable to use the ones from the git-repository?

I see this PR actually to enable SSL on a zero effort basis. So I don't expect anyone doing manual work for SSL while using this container.

So what do you think: Is it desirable to have every mailcow user running with the same dhparams?